pkg/cacheutil/memcached_client.go

```diff
@@ -414,6 +416,7 @@ func (c *memcachedClient) enqueueAsync(op func()) error {
 	case c.asyncQueue <- op:
 		return nil
 	default:
+		c.skipped.WithLabelValues(opSet, reasonAsyncBufferFull).Inc()
```

Comment:
In case of this metric.. can we just drop `level.Warn(limitedLogger).Log("msg", "failed to store one or more items into memcached", "failed", failed, "firstErr", firstErr)`?

It sounds to me like metric is enough.

Reply:
This is again why handling error multiple times is tricky as you can by accident replicate this error many times (we inc metric and return err in same time)